pages/place/routers/confirm.js

```diff
@@ -71,7 +71,7 @@ module.exports = settings => {
       if (req.session.form && req.session.form[req.model.id]) {
         return next();
       }
-      return res.redirect(req.buildRoute(settings.page));
+      return res.redirect(req.buildRoute('place.delete'));
```

Comment:
This looks like it possibly needs reverting.

Reply:
was just testing you were awake 😄